src/vivarium/framework/values.py

```diff
@@ -471,7 +479,9 @@ def get_value(self, name: str) -> Pipeline:
             (frequently just a :class:`pandas.Index` representing the
             simulants).
         """
-        return self._pipelines[name]  # May create a pipeline.
+        pipeline = self._pipelines.get(name) or Pipeline(name)
+        self._pipelines[name] = pipeline
```

Comment:
Feels weird to (potentially) get a dict value and then turn right around and re-assign it to the same key (but I also like not adding if/else logic so maybe it's fine?)

Reply:
Yeah I thought about that for a bit as well. I ultimately decided that I preferred the succinctness of the reassignment, which I think is more readable as well. Since the behavior is identical and the reassignment isn't exactly expensive, I did it this way.